Gems/Atom/Feature/Common/Assets/Materials/Special/ShadowCatcher.azsl

```diff
@@ -15,7 +15,7 @@ option enum class OpacityMode
     Blended, 
     TintedTransparent
     
-} o_opacity_mode = OpacityMode::Opaque;
+} o_opacity_mode = OpacityMode::Blended;
```

Comment:
is this a debugging change? Doesn't seem related to the shadow enabled change.

Reply:
This is "Fixed ShadowCatcher shader" mentioned in the description.